bundle_templates/item_template.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "{id}",
+    "fileName": "{fileName}",
+    "version": "{version}",
+    "description": "{description}",
+    "requiredVersion": "{requiredVersion}",
```

Comment:
Do you remove field if variable is not defined?

Reply:
is it optional?

Reply:
yes. both fields, `requiredVersion` and `minimalVersion` are optional.